ansible/roles/etesync/templates/etc/etesync-server/etesync-server.ini.j2

```diff
@@ -0,0 +1,16 @@
+# {{ ansible_managed }}
+
+[global]
+secret_file = {{ etesync__config_secret_key_filepath }}
+debug = false
+
+[allowed_hosts]
+{% for host in ([ etesync__config_allowed_hosts ]
+				if etesync__config_allowed_hosts is string
+				else etesync__config_allowed_hosts) %}
+allowed_host{{ loop.index }} = {{ host }}
+{% endfor %}
+
+[database]
+engine = django.db.backends.sqlite3
+name = {{ etesync__data }}/db.sqlite3
```

Comment:
I'm suprised that the role does not support PostgreSQL as an alternative to SQLite. I suppose that this can be added later and backported to a stable release.

Reply:
Jup. The only reasons I did not add PostgreSQL support is that for my small deployment I decided to go with SQLite. If someone needs PostgreSQL, it can be added.

Reply:
I would like to request postgres as my backup solution can't really deal with sqlite.

Reply:
In that case I suggest that the PR will be merged after `v2.0.0` release which is imminent (perhaps today). I'll revert the `etckeeper` change as well.

Reply:
@SuperSandro2000 Feel free to add PostgreSQL support.